### PR TITLE
add common package

### PR DESCRIPTION
**Pull request by hougangliu**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/491)

### CODE REVIEW

File: pkg/common/v1alpha1/common.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package pkg
+package v1alpha1
```
Comment (andreyvelich) [inline pkg/common/v1alpha1/common.go]: Maybe name this package `common` ?

Comment (hougangliu) [reply pkg/common/v1alpha1/common.go]: based on go best practices, package name should keep same as its directory name. all Katit package follows this style.

Comment (gaocegege) [reply pkg/common/v1alpha1/common.go]: Yeah, I think we should keep v1alpha1

**Pull request merged**